spinalcordtoolbox/scripts/sct_dmri_moco.py

```diff
@@ -135,7 +137,37 @@ def get_parser():
         choices=[0, 1, 2],
         default=1,
         # Values [0, 1, 2] map to logging levels [WARNING, INFO, DEBUG], but are also used as "if verbose == #" in API
-        help="Verbosity. 0: Display only errors/warnings, 1: Errors/warnings + info messages, 2: Debug mode")
+        help="Verbosity. 0: Display only errors/warnings, 1: Errors/warnings + info messages, 2: Debug mode"
+    )
+    optional.add_argument(
+        '-qc',
+        metavar=Metavar.folder,
+        action=ActionCreateFolder,
+        help="The path where the quality control generated content will be saved."
+    )
+    optional.add_argument(
+        '-qc-seg',
+        metavar=Metavar.file,
+        help="Segmentation of spinal cord to improve cropping in qc report"
+    )
```

Comment:
Right now, it's not very clear that `-qc-seg` is mandatory for `-qc`. It might be good to add a note about this in the help descriptions.

When I tried `-qc` for the first time, I only found out `-qc-seg` was necessary via the error message, which felt a bit jarring.

Reply:
> Right now, it's not very clear that -qc-seg is mandatory for -qc. It might be good to add a note about this in the help descriptions.

+1